### PR TITLE
indexing: add a custom text analyzer in ES template

### PR DESCRIPTION
**Pull request by Garfield-fr**
Adds a sharded lowercase and asciifolding analzer using ES templates to be used in several indexes.

How to test:
- `script/bootstrap`
- `script/setup`
-  Kibana console: `GET /_template` to show custom analyzer 

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

### CODE REVIEW

File: rero_ils/modules/cli.py [hunk 1]
```diff
@@ -159,3 +162,23 @@ def schedules():
...
         click.echo(key + '\t', nl=False)
         click.echo(value)
+
+
+@utils.command()
+@click.option('--force', is_flag=True, default=False)
+@with_appcontext
+@es_version_check
+def init(force):
+    """Initialize registered templates, aliases and mappings."""
```
Comment (jma) [inline rero_ils/modules/cli.py]: As it is a invenio patch. You can explain why it is needed and you can also add a TODO for future removal, eventually linked to an issue number on `invenio-search`.

File: rero_ils/es_templates/__init__.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, RERO does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Records Templates Elasticsearch."""
+
+def list_es_templates():
```
Comment (jma) [inline rero_ils/es_templates/__init__.py]: documentation is missing.

File: rero_ils/es_templates/v6/record.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+  "index_patterns": "*",
+  "settings": {
+    "analysis": {
+      "analyzer": {
+        "g_lowercase_asciifolding": {
```
Comment (jma) [inline rero_ils/es_templates/v6/record.json]: what `g_` does it means?

Comment (Garfield-fr) [reply rero_ils/es_templates/v6/record.json]: for global ?

**Comment by jma** [comment]
some suggestions about the commit message:
- Template does not sounds like a component
- you can explain which template the commit contains (here probably analyzer only).

File: rero_ils/modules/cli.py (hunk 1 above)
Comment (Garfield-fr) [reply rero_ils/modules/cli.py]: done

File: rero_ils/es_templates/__init__.py (hunk 2 above)
Comment (Garfield-fr) [reply rero_ils/es_templates/__init__.py]: done

File: rero_ils/es_templates/v6/record.json (hunk 3 above)
Comment (Garfield-fr) [reply rero_ils/es_templates/v6/record.json]: g_ to global_

**Pull request merged**